lib/ProductOpener/Display.pm

```diff
@@ -6519,7 +6519,8 @@ sub display_field($$) {
 
 
 	if ((defined $value) and ($value ne '')) {
-		if (($field eq 'link') and ($value =~ /^http/)) {
+		# See https://stackoverflow.com/a/3809435
+		if (($field eq 'link') and ($value =~ /[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/)) {
```

Comment:
We should have http or https in most values. If we don't have it (e.g something like www.someproducer.com), then we need to add http:// otherwise the link after will not work.